### PR TITLE
raf: add createMs primitive

### PR DESCRIPTION
**Pull request by atk**
I recently animated some SVGs and found the pattern helpful to create a milliseconds counter using createRAF and targetFPS, so I want to make this pattern an official primitive.

### CODE REVIEW

**Review by thetarnav — COMMENTED** [review]
I like that.
You could probably add a "time in miliseconds since the initial frame, incremented every frame (with given framerate)" explanation.
Simply saying "a reactive miliseconds signal" is a bit vague.
I had to read the implemetation and stare at the example a while to get it.
Also, how would one go about reseting the animation? Probably wrapping in a memo, but worth showing that.

File: packages/raf/README.md [hunk 1]
```diff
@@ -84,6 +84,30 @@ function targetFPS(
...
+## createMs
+
+Using createRAF and targetFPS to create a reactive milliseconds counter with configurable frame rate.
+
+It takes the framerate as single argument, either as `number` or `Accessor<number>`.
+
+```ts
+import { createMs } from "@solid-primitives/raf";
+
+const ms = createMs(60);
```
[the remark this answers is not in the document]
Comment (alex-crabnebula) [reply packages/raf/README.md:96]: Right, it should be used inside a component.

**Comment by alex-crabnebula** [comment]
> Also, how would one go about reseting the animation? Probably wrapping in a memo, but worth showing that.

Good point. I should add both a reset() method and a maxFrames option for resetting the ms counter.

**Review by thetarnav — APPROVED** [review]
looks decent

**Pull request merged**